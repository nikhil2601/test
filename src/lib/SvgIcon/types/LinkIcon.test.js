import React from 'react';
import { shallow } from 'enzyme';

import LinkIcon from './LinkIcon';

describe('<LinkIcon />', () => {
    const wrapper = shallow(<LinkIcon />);

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
