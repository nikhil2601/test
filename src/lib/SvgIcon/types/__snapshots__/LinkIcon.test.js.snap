// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinkIcon /> should render correctly 1`] = `
<SvgIcon
  color="inherit"
  height="1rem"
  margin={null}
  opacity={null}
  padding={null}
  titleAccess={null}
  viewBox="0 0 18 18"
  width="1rem"
>
  <path
    d="M13.05 16.11l-3.549-3.55.977-.976.594.595a.78.78 0 0 0 1.107 0 .783.783 0 0 0 0-1.107l-.595-.595.977-.976 3.55 3.55-3.06 3.06zM5.44 8.5L1.89 4.95l3.06-3.06L8.5 5.44l-.978.977-.594-.595A.783.783 0 0 0 5.82 6.928l.595.594-.977.977zm12.33 3.999L13.115 7.84a.783.783 0 0 0-1.107 0l-1.53 1.53L8.63 7.522l1.53-1.53a.783.783 0 0 0 0-1.106L5.503.229a.782.782 0 0 0-1.107 0L.23 4.395a.783.783 0 0 0 0 1.107l4.657 4.657a.78.78 0 0 0 1.107 0l1.53-1.53 1.848 1.849-1.53 1.53a.783.783 0 0 0 0 1.106l4.656 4.657a.783.783 0 0 0 1.107 0l4.167-4.167a.783.783 0 0 0 0-1.106z"
    fillRule="evenodd"
  />
</SvgIcon>
`;
